tests/python/test_f16.py

```diff
@@ -278,3 +278,15 @@ def foo():
 
     foo()
     assert (f[0] == approx(f[1], rel=1e-3))
+
+
+@pytest.mark.parametrize('dtype', [ti.f32, ti.f64])
+@ti.test(arch=archs_support_f16)
+def test_cast_to_f16(dtype):
+    @ti.kernel
+    def func() -> ti.f16:
+        a = ti.cast(23.0, ti.f64)
+        b = ti.cast(4.0, ti.f64)
```

Comment:
Is it intended to be
```suggestion
        a = ti.cast(23.0, dtype)
        b = ti.cast(4.0, dtype)
```
so that the `dtype` argument can be used?

Reply:
Thanks!